tests/utils/check_code.sh

```diff
@@ -133,6 +138,39 @@ if [[ "$test" == "bandit" || "$test" == "all" ]]; then
     echo "TEST COMPLETE bandit"
     retval=$((bandit_retval+retval))
 fi
+if [[ "$test" == "pytest" || "$test" == "all" ]]; then
+    echo ""
+    echo "CALLING TEST:"
+
+    # First test the geoips package pytests - these include pytests of any other
+    # installed plugin packages, so always test geoips repo pytests.
+    echo "pytest --cov=geoips -c $CONFIG_PATH/pytest.ini $GEOIPS_PACKAGES_DIR/geoips/tests"
+    pytest --cov=geoips -c $CONFIG_PATH/pytest.ini $GEOIPS_PACKAGES_DIR/geoips/tests
+    pytest_geoips_retval=$?
+
+    # Only test the current path if it is NOT geoips.  Don't test geoips twice.
+    if [[ "$pkgname" != "geoips" ]]; then
+        echo "pytest --cov=$pkgname -c $CONFIG_PATH/pytest.ini $path/tests"
+        pytest --cov=$pkgname -c $CONFIG_PATH/pytest.ini $path/tests
+        pytest_pkg_retval=$?
+
+        # Return value of 5 indicates that no tests were run.
+        # https://docs.pytest.org/en/7.1.x/reference/exit-codes.html
+        # If no tests were run, still return 0.  Not all packages have
+        # individual pytests defined, and we don't want a failure due to that.
+        if [[ "$pytest_pkg_retval" == "5" ]]; then
+            pytest_pkg_retval="No tests found"
+        else
+            # Add in the pkg pytest retval
+            retval=$((pytest_pkg_retval+retval))
+        fi
+    else
+        pytest_pkg_retval="Not tested"
+    fi
+    echo "TEST COMPLETE pytest"
+    # Now add in the geoips pytest retval
+    retval=$((pytest_geoips_retval+retval))
+fi
```

Comment:
Slight modifications here. 

  black return: 0
  flake8 return: 0
  bandit return: 0
  pytest geoips return: 0
  pytest geoips return: Not tested
  interfaces return: 0

  black return: 0
  flake8 return: 0
  bandit return: 0
  pytest geoips return: 0
  pytest recenter_tc return: No tests found
  interfaces return: 0